daml-script/dump/src/main/scala/com/daml/script/dump/Encode.scala

```diff
@@ -67,7 +70,7 @@ private[dump] object Encode {
       (Doc.text("dump Parties{..} = do") /
         Doc.stack(
           sortedAcs.map(createdEvent =>
-            encodeCreatedEvent(partyMap, cidMap, cidRefs, createdEvent)
+            encodeSubmitCreatedEvents(partyMap, cidMap, cidRefs, Seq(createdEvent))
```

Comment:
We could change this to issue a single `submit` that issues all `createCmd`s to reconstruct the ACS in one go.
However, with a large ACS this might require a pretty large tuple to bind all the contract Ids. I'm not sure if this could cause issues.